src/Orleans/Cancellation/GrainCancellationTokenSource.cs

```diff
@@ -1,14 +1,14 @@
-﻿using System;
-using System.Threading.Tasks;
-using Orleans.CodeGeneration;
+﻿using Orleans.CodeGeneration;
 using Orleans.Serialization;
+using System;
+using System.Threading.Tasks;
```

Comment:
Yes, some option in my Visual Studio was off. Sorting namespaces with System first.